packages/portal-proto/src/features/cDave/Controls.tsx

```diff
@@ -144,8 +144,11 @@ const FieldControl: React.FC<FieldControlProps> = ({
             <label
               className="font-content font-medium text-md"
               htmlFor={`switch-${field.full}`}
+              aria-labelledby={`switch-${field.full}`}
             >
-              <Highlight highlight={searchTerm}>{displayName}</Highlight>
+              <span id={`switch-${field.full}`}>
+                <Highlight highlight={searchTerm}>{displayName}</Highlight>
+              </span>
```

Comment:
revert this, old code looked to be more correct, dont know if you can target an id in a Switch this way though

Reply:
line 204 has htmlFor={`switch-${field.full}`} i dont know if two elements can have the same target

Reply:
reverted the code, good catch thank you